tests/e2e/e2e_setup_test.go

```diff
@@ -484,12 +520,46 @@ func (s *IntegrationTestSuite) upgrade() {
 		}
 	}
 
-	// remove all containers so we can upgrade them to the new version
 	for _, chainConfig := range s.chainConfigs {
 		s.upgradeContainers(chainConfig, chainConfig.propHeight)
 	}
 }
 
+func (s *IntegrationTestSuite) upgradeFork() {
+
+	for _, chainConfig := range s.chainConfigs {
+
+		for i := range chainConfig.validators {
+			if _, ok := chainConfig.skipRunValidatorIndexes[i]; ok {
+				continue
+			}
+
+			validatorResource, exists := s.containerManager.GetValidatorResource(chainConfig.meta.Id, i)
+			require.True(s.T(), exists)
+			containerId := validatorResource.Container.ID
+			containerName := validatorResource.Container.Name[1:]
+
+			s.T().Logf("waiting to reach fork height on %s validator container: %s", containerName, containerId)
+			s.Require().Eventually(
+				func() bool {
+					currentHeight := s.getCurrentChainHeight(chainConfig, i)
+					if currentHeight < s.forkHeight {
+						s.T().Logf("current block height on %s is %v, waiting for block %v container: %s", containerName, currentHeight, s.forkHeight, containerId)
+						return false
+					}
+					if currentHeight > s.forkHeight {
+						return true
+					}
+					return true
```

Comment:
Seems that this is true either way
```suggestion
					return true
```